1-documentation/doc-inputs/doc-inputs.html

```diff
@@ -131,6 +131,84 @@ <h2> Input with error </h2>
+                    Stars & above</label>
+                <label for="one-stars"> <input type="radio" id="one-stars" name="one-stars" value="one-stars"> 1
+                    Stars & above</label>
+                <label for="any-stars"> <input type="radio" id="any-stars" name="any-stars" value="any-stars">
+                    Any
+                    Rating</label>
+            </section>
+        </div>
+
+        <div class="embed-wrapper">
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=677&ds=true&dsyoff=21px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522product-filters%2522%253E%250A%2520%2520%253Ch3%253E%2520Rating%2520%253C%252Fh3%253E%250A%2520%2520%253Csection%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522four-stars%2522%253E%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522four-stars%2522%2520name%253D%2522four-%2509%2509stars%2522%2520value%253D%2522four-stars%2522%253E%25204%250A%2520%2520%2520%2520%2509%2509Stars%2520%2526%2520above%253C%252Flabel%253E%250A%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522three-stars%2522%253E%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522three-stars%2522%2520%2509%2509%2509%2509%2509name%253D%2522three-stars%2522%2520value%253D%2522three-stars%2522%253E%25203%250A%2520%2520%2520%2520%2509%2509Stars%2520%2526%2520above%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522two-stars%2522%253E%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522two-stars%2522%2520name%253D%2522two-%2509%2509%2509stars%2522%2520value%253D%2522two-stars%2522%253E%25202%250A%2520%2520%2520%2520%2509%2509Stars%2520%2526%2520above%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522one-stars%2522%253E%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522one-stars%2522%2520name%253D%2522one-%2509%2509%2509stars%2522%2520value%253D%2522one-stars%2522%253E%25201%250A%2520%2520%2520%2520%2509%2509Stars%2520%2526%2520above%253C%252Flabel%253E%250A%2520%2520%2520%2520%253Clabel%2520for%253D%2522any-stars%2522%253E%2520%253Cinput%2520type%253D%2522radio%2522%2520id%253D%2522any-stars%2522%2520name%253D%2522any-%2509%2509%2509stars%2522%2520value%253D%2522any-stars%2522%253E%250A%2520%2520%2520%2520%2509%2509Any%2520Rating%253C%252Flabel%253E%250A%2520%2520%253C%252Fsection%253E%250A%253C%252Fdiv%253E"
+                style="width: 677px; height: 448px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+
+
+        <h2> Radio Button 2 </h2>
+        <div class="product-filters">
+            <h3> Sort by </h3>
+            <section>
+                <label for="low-to-high"> <input type="radio" id="low-to-high" name="low-to-high" value="low-to-high">
```

Comment:
Similarly here you can give `name` attribute as `sort-by-price` to allow user to select only one radio button among the 2.